server/grpc/server_test.go

```diff
@@ -30,7 +31,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.network = network.New(s.T(), network.DefaultConfig())
 	s.Require().NotNil(s.network)
 
-	_, err := s.network.WaitForHeight(2)
+	_, err := s.network.WaitForHeightWithTimeout(2, 60*time.Second)
```

Comment:
This is still another race condition. Now we know we've reached height 2, but we don't know we haven't reached height 3... right? So let's just not expect a specific height. I think that's the fix. We don't know the height it's non-deterministic.

Reply:
yeah, you're right